src/stream.h

```diff
@@ -112,14 +112,18 @@ namespace librealsense
         std::shared_ptr<stream_profile_interface> clone() const override
         {
             auto res = std::make_shared<video_stream_profile>(platform::stream_profile{});
+            if( res == nullptr )
+                throw librealsense::invalid_value_exception( "Stream profiles failed in clone." );
             auto id = environment::get_instance().generate_stream_id();
+
             res->set_unique_id( id );
             LOG_DEBUG( "video_stream_profile::clone, id= " << id );
-            res->set_dims(get_width(), get_height());
```

Comment:
Let's try to avoid format changes for lines we do not change, it will keep the PR focused.

Reply:
You are right. My new clang-format file changes lines and it looks different from the old format.